src/docs/reference/modules/Resources/README.md

```diff
@@ -232,14 +236,14 @@ You can append a version hash that will be calculated, and calculation cached, a
 
 ##### Specify location
 
-By default all scripts are rendered in the footer. You can override it like this:
+By default all scripts are rendered whereever the `FootScript` helper is located [See Foot Resources](#foot-resources). You can override it like this:
```

Comment:
whereever => wherever

Reply:
Thanks @agriffard - I changed the wording here based on the outcome of #6740